CIP-XXXX/README.md

```diff
@@ -0,0 +1,139 @@
+
+We define a byte format to combine the tx ID and the index to form a single valid byte string, as such it can be converted into a hex format and have its own bech32 prefix.
+
+Transaction ID is always a 32 byte length, hence we can append the index bytes to the tx id, please see below
+
+
+TX ID in Hex - `0000000000000000000000000000000000000000000000000000000000000000`
+Gov Action index in HEX - `11` (number 17)
+
+gov action ID - `000000000000000000000000000000000000000000000000000000000000000011`
+
+bech32 - `gov_action1qqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqpzklpgpf`
+
+## Bech32 Encoding
+
+| Prefix        | Meaning                                                 | Contents                                                          |
+| ------------- | --------------------------------------------------------| ----------------------------------------------------------------- |
+| `drep`        | DRep identifier                                         | DRep Credential                                                   |
+| `cc_hot`      | CC Hot identifier                                       | CC Hot Credential                                                 |
+| `cc_cold`     | CC Cold identifier                                      | CC Cold Credential                                                |
+| `gov_action`  | gov action identifier                                   | gov action tx id concatenated with index                          |
+
+
+### Identifier Test Vector
+
+We can define a complete identifier as per the spec above by combining the header and the key, see below
+
+#### Constitutional Committee Hot Credential
+
+Key - `00000000000000000000000000000000000000000000000000000000`
+Type - `Key Hash`
+
+Identifier - `0200000000000000000000000000000000000000000000000000000000`
+
+Bech32 - `cc_hot1qgqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqvcdjk7`
+
+#### Constitutional Committee Cold Credential
+
+Key - `00000000000000000000000000000000000000000000000000000000`
+Type - `Script Hash`
+
+Identifier - `1300000000000000000000000000000000000000000000000000000000`
+
+Bech32 - `cc_cold1zvqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqq6kflvs`
+
+#### DRep Credential
+
+Key - `00000000000000000000000000000000000000000000000000000000`
+Type - `Key Hash`
+
+Identifier - `2200000000000000000000000000000000000000000000000000000000`
+
+Bech32 - `drep1ygqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqq7vlc9n`
+
+## Rationale: how does this CIP achieve its goals?
+
+By introducing a header and payload design for different keys, we can achieve specifying credentials in a hex format (bytes), being able to share and use across tooling is possible. We also specify bech32 prefixes for user friendly identification of the identifiers.
+
+
+## Path to Active
+
+### Acceptance Criteria
+
+- [] implemented by various tools
```

Comment:
```suggestion
### Acceptance Criteria

- [ ] Implemented by various tools

### Implementation Plan
```
At the very least it also needs an _Implementation Plan_ and more detail across **both** subheadings.  Checkboxes appear properly formatted when they have either a tick-character (`x` or `X`) or a blank space inside them. :)